RFC-001-BAM.md

```diff
@@ -427,16 +394,17 @@ This section defines how headers are encoded in the JSON-based text encoding.
 Let's start off with an example:
 
 ```json
-"_source_ledger" : {
-    "value": "Bitcoin",
-    "parameters": {
-        "network": "mainnet"
-    }
+"_payment_method" : {
+  "value": "credit-card",
+  "parameters": {
+    "provider": "visa",
```

Comment:
```suggestion
    "provider": "tenx",
```
I am not keen to have too much references to the old financial system in our repo ;)